test/webpack.config.js

```diff
@@ -0,0 +1,24 @@
+const path = require(`path`);
+
+const webpackConfig = {
```

Comment:
not sure if you care about verbosity of webpack but I always find it annoying that it pastes a huge chunk.

I like to add

```
  stats: {
    modules: false,
    children: false,
    chunks: false,
    assets: true,
    assetsSort: `name`,
    timings: true,
  },
```

Reply:
aight https://github.com/mixpanel/webcomponent/pull/5/commits/8b10196415b5d30e7f7d1f53c9faebd9dede1759